src/main/java/de/letsdev/maven/plugins/ios/Utils.java

```diff
@@ -351,4 +423,82 @@ public static XcodeArchiveProductType getExportProductType(String productName) {
 
         return type;
     }
+
+    public static boolean shouldUseWorkspaceFile(Map<String, String> properties) {
+
+        if (properties.containsKey(PLUGIN_PROPERTIES.COCOA_PODS_ENABLED.toString())) {
+            return properties.get(PLUGIN_PROPERTIES.COCOA_PODS_ENABLED.toString()).equals("true");
+        }
+
+        return false;
+    }
+
+    public static String getXcprettyCommand(String logFileName, String jsonOutputFile) {
+
+        String xcPrettyStatement = "";
+        if (jsonOutputFile != null && !jsonOutputFile.isEmpty()) {
+            xcPrettyStatement += "| XCPRETTY_JSON_FILE_OUTPUT=" + jsonOutputFile;
+        } else {
+            xcPrettyStatement += " | tee " + logFileName + " |";
+        }
+        xcPrettyStatement += " xcpretty";
+        if (jsonOutputFile != null && !jsonOutputFile.isEmpty()) {
+            xcPrettyStatement += " -f `xcpretty-json-formatter`";
+        }
+        xcPrettyStatement += " && exit ${PIPESTATUS[0]}";
+        return xcPrettyStatement;
+    }
+
+    public static void executeShellScript(String scriptName, String value1, String value2, String value3,
+                                          File workDirectory) throws IOSException {
+
+        // Run shell-script from resource-folder.
+        try {
+            File tempFile = File.createTempFile(scriptName, "sh");
+
+            InputStream inputStream = ProjectBuilder.class.getResourceAsStream("/META-INF/" + scriptName);
+            OutputStream outputStream = new FileOutputStream(tempFile);
+
+            byte[] buffer = new byte[1024];
+            int bytesRead;
+
+            while ((bytesRead = inputStream.read(buffer)) != -1) {
+
+                outputStream.write(buffer, 0, bytesRead);
+            }
+
+            outputStream.close();
+
+            if (value1 == null) {
+                value1 = "";
+            }
+
+            if (value2 == null) {
+                value2 = "";
+            }
+
+            if (value3 == null) {
+                value3 = "";
+            }
+
+            ProcessBuilder processBuilder = new ProcessBuilder("sh", tempFile.getAbsoluteFile().toString(), value1,
+                    value2, value3);
+
+            processBuilder.directory(workDirectory);
+            CommandHelper.performCommand(processBuilder);
+        } catch (IOException e) {
+            e.printStackTrace();
+            throw new IOSException(e);
+        }
+    }
+
+    public static String createJsonOutputFilePath(String relativeFilePath, Map<String, String> properties) {
```

Comment:
aktuell geben wir bei allen Aufrufen das build/reports noch mit rein. Das kannst du entfernen und hier in der Methode dann vor den relativeFilePath anfügen. Dann muss man bspw. nur noch results-clean.json übergeben